packages/assets/src/tools/validate-files.ts

```diff
@@ -30,14 +58,27 @@ async function validateValidOrgs() {
     const diff = difference(expectedFiles.files, orgAssets.files);
 
     if (diff.length > 0) {
-      hasErrors = true;
-      console.error(
-        `
-${themeVariantAsString(orgAssets.org)} is missing some assets:
-${diff.join('\n')}
-
-`,
+      console.warn(
+        `${themeVariantAsString(
+          orgAssets.org,
+        )} seems to be missing some assets, checking if common files can be used for these files:\n${diff.join(
+          '\n',
+        )}`,
       );
+      const filesMissingBothInOrgAndCommon =
+        await verifyThatMissingOrgFilesHasCommonReplacements(diff);
+
+      if (filesMissingBothInOrgAndCommon.length) {
+        hasErrors = true;
+        console.error(
+          `
+                ${themeVariantAsString(
```

Comment:
Won't this printout generate some serious whitespace? Either de-indent block (preferred) or use something like `noindent`.